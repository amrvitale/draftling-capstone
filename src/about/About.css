body, html {
    height: 100%;
    margin: 0;
}

.aboutHero {
    background-image: url('./penNotebook.jpg');
    height: 100%; 
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
}

.aboutHeader{
    color: #35322a;    
    font-family: 'Playfair Display', 'Georgia', serif;
}

p, ul, h2 {
    color: whitesmoke;
}

.fictionLengths{
    font-size: 30px;
    text-align: center;
}

.feedbackOptions {
    font-size: 40px;
}