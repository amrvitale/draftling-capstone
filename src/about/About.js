import React from 'react';
import './About.css'

class About extends React.Component {
    render() {
        return (
            <div className="aboutDetails">
                <div className="aboutHero">
                    <h2 className="aboutHeader">About Draftling</h2>
                </div>
                    <p>Draftling is a forum for getting and giving feedback on your fiction writing. Post your writing and receive indepth feedback from fellow writers and editors! We accept the following types of fiction pieces: </p>
                    <ul>
                       <li>Microfiction</li>
                        <li>Flash Fiction</li>
                        <li>Short Story</li>
                        <li>Novellete</li>
                        <li>Novella</li>
                        <li>Novel</li>
                    </ul>
                
                    <h2>Feedback Options</h2>
                    <p>Beta readers can provide feedback via comments or questionnaires.  To get feedback on your piece, set your feedback availability to one of these options, and readers will post their clear, concise, and useful feedback to help you improve your writing!</p>
            </div>
        );
    }
}

export default About;