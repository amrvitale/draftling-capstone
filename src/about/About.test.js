import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import About from './About';

describe (`About component`, () => {
    it('renders a .About by default', () => {
        const wrapper = shallow(<About />)
        expect (toJson(wrapper)).toMatchSnapshot()
    })

    
})