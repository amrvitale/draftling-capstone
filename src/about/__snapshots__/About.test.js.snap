// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders a .About by default 1`] = `
<div
  className="aboutDetails"
>
  <div
    className="aboutHero"
  >
    <h2
      className="aboutHeader"
    >
      About Draftling
    </h2>
  </div>
  <p
    className="aboutPara"
  >
    Draftling is a forum for getting and giving feedback on your fiction writing. Post your writing and receive indepth feedback from fellow writers and editors! We accept the following types of fiction pieces: 
  </p>
  <br />
  <ul
    className="fictionLengths"
  >
    <li>
      Microfiction
    </li>
    <li>
      Flash Fiction
    </li>
    <li>
      Short Story
    </li>
    <li>
      Novellete
    </li>
    <li>
      Novella
    </li>
    <li>
      Novel
    </li>
  </ul>
  <br />
   
  <br />
    
  <br />
  <h2
    className="feedbackOptions"
  >
    Feedback Options
  </h2>
  <p
    className="aboutPara"
  >
    Beta readers can provide feedback via comments or questionnaires.  To give feedback on a piece, click the Draftling and select one of these options. The best readers post clear, concise, and useful feedback to help you improve your writing!
  </p>
</div>
`;
